packages/sqs-batch/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@middy/sqs-batch",
+  "version": "1.0.0-beta",
+  "description": "SQs batch error handler middleware for the middy framework",
+  "engines": {
+    "node": ">=10"
+  },
+  "engineStrict": true,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "test": "npm run test:typings && npm run test:unit",
+    "test:unit": "jest",
+    "test:typings": "typings-tester --config tsconfig.json index.d.ts"
+  },
+  "license": "MIT",
+  "keywords": [
+    "Lambda",
+    "Middleware",
+    "Serverless",
+    "Framework",
+    "AWS",
+    "AWS Lambda",
+    "Middy",
+    "SQS",
+    "Batch",
+    "Error"
+  ],
+  "author": {
+    "name": "Middy contributors",
+    "url": "https://github.com/middyjs/middy/graphs/contributors"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/middyjs/middy.git"
+  },
+  "bugs": {
+    "url": "https://github.com/middyjs/middy/issues"
+  },
+  "homepage": "https://github.com/middyjs/middy#readme",
+  "peerDependencies": {
+    "@middy/core": ">=1.0.0-alpha",
+    "aws-sdk": ">=2.221.1"
+  },
+  "devDependencies": {
+    "@middy/core": "^1.0.0-beta",
+    "@serverless/event-mocks": "^1.1.1",
+    "promise.allsettled": "^1.0.2"
+  },
+  "gitHead": "7a6c0fbb8ab71d6a2171e678697de9f237568431",
+  "dependencies": {
+    "jest": "^25.1.0"
```

Comment:
Is this supposed to be a dependency?

Reply:
`devDependencies` makes more sense to me, I'm guessing it was copied from somewhere.

Reply:
Absolutely not. Good catch.